.github/workflows/pull_requests.yml

```diff
@@ -6,6 +6,8 @@ on:
 
 env:
   NODE_VERSION: 16
+  ACCESS_TOKEN_SECRET: "ZzpPypwUKQiCLra6h72ny8DemduUwQbg"
```

Comment:
Humm what is this for ? Should we really push secret visible like this ?

Reply:
yes c'est pas important, c'est juste utilisé par la CI donc pas spécialement de problème avec ça (dépendant de l'environnement ;) )